README.md

```diff
@@ -22,7 +22,7 @@ $ sbt package
 You can additionally run unit test through:
 
 ```
-$ sbt unitTest
+$ sbt test -Dconfig.file=src/main/resources/reference.conf
```

Comment:
this should be `src/test` ?

It also shouldn't be necessary, based on what's in build.sbt. That may take some investigation to see why your environment needs it.